src/Button.js

```diff
@@ -2,18 +2,25 @@
 import React from 'react';
 
 function resetProduct() {
-  document.getElementById('productLogo').src='dist/img/DPW_Skull_Logo_Mask.jpg';
+  	document.getElementById('productLogo').src='img/DPW_Skull_Logo_Mask.jpg';
+
+	document.getElementById("eyesBlue").checked = false;
+	document.getElementById("eyesGreen").checked = false;
+	document.getElementById("eyesGrey").checked = false;
+		
 }
 
 const Button = (props) => {
 	//list multiple props here
-	const { type = '',
-	onClick,
-	propThree,
-	propFour } = props
+	const { 
+		type = 'button',
+		onClick = resetProduct,
+		propThree,
+		propFour
```

Comment:
Add a children prop in here so you can change the label of each button you render and this will allow your button component to be reusable. Your props object should look something like this:

`const {
    type = 'button',
    onClick,
    children
}`

Then, in where the button is returned, update it so it uses the children prop:

`<button type={type} onClick={onClick}>{children}</button>`

So, when you import your button component and render it in your HTML, you can pass any label into your button:

`<Button>Login</Button>`

Reply:
Great shout!

Reply:
Thanks @justtoconfirm this makes sense too and I've actioned and committed these three points